remediation/code-samples/03-concat.js

```diff
@@ -26,6 +26,7 @@
  let nickels = [5,5];
  let dimeNickel = ['10','10','5','5'];
 
+ // PLEASE REVIEW
```

Comment:
Great solution! Converting the two arrays to Strings is one way we can concatenate them. But the `join()` function is not absolutely necessary to do this.

For example,
```
let left = leftArray.join(",");
let right = rightArray.join(",");
let together = "";
together = left + "," + right;
```

could actually be replaced with

`let together = leftArray + "," + rightArray;`